apps/web-giddh/src/app/shared/create-address/create-address.component.ts

```diff
@@ -83,6 +91,11 @@ export class CreateAddressComponent implements OnInit, OnDestroy {
      */
     public ngOnInit(): void {
         this.setFormData();
+        this.store.pipe(select(state => state.session.activeCompany), takeUntil(this.destroyed$)).subscribe(activeCompany => {
```

Comment:
use component store

Reply:
Sir, here no component store is implemented that's why I use global store.